naresh/src/main/java/com/lftechnology/phpjava/ems/constants/Constant.java

```diff
@@ -0,0 +1,83 @@
+package com.lftechnology.phpjava.ems.constants;
+
+
+/**
+ * Constant
+ *
+ * @author Naresh Maharjan <nareshmaharjan@lftechnology.com>
+ * @since August, 09 2016
+ */
+public class Constant {
+    public static final int ADD_NEW_USER = 1;
+    public static final int UPDATE_PROFILE = 1;
+    public static final int SEARCH_USER = 2;
+    public static final int TERMINATE_USER = 3;
+    public static final int DELETE_USER = 4;
+    public static final int EXIT = 0;
+    public static final String PROPERTY_URL = "url";
+    public static final String PROPERTY_DRIVER = "driver";
+    public static final String PROPERTY_USERNAME = "username";
+    public static final String PROPERTY_PASSWORD = "password";
+    public static final String POST_LOGIN_ACTION = "postLogin";
+    public static final String ADD_USER_ACTION = "addUser";
+    public static final String SEARCH_USER_ACTION = "searchUser";
+    public static final String TERMINATE_USER_ACTION = "terminateUser";
+    public static final String DELETE_USER_ACTION = "deleteUser";
+    public static final String EXIT_ACTION = "exit";
+    public static final String UPDATE_PROFILE_ACTION = "updateProfile";
+    public static final String TERMINATE_USER_MESSAGE = "3. Terminate User";
+    public static final String ADD_NEW_USER_MESSAGE = "1. Add New User";
+    public static final String SEARCH_USERS = "2. Search User(s)";
+    public static final String DELETE_USER_MESSAGE = "4. Delete User";
+    public static final String EXIT_MESSAGE = "0. Exit";
+    public static final String UPDATE_PROFILE_MESSAGE = "1. Update Profile";
+    public static final String CHOOSE_ONE_OF_THE_FOLLOWING_USER_TYPE = "Choose One Of The Following User Type";
+    public static final String ADD_ADMIN_USER = "1. Add Admin User";
+    public static final String ADD_NORMAL_USER = "2. Add Normal User";
+    public static final String EMPLOYEE = "employee";
+    public static final String COMMON_UTILITY = "commonUtility";
+    public static final String ENTER_USER_DETAILS = "Enter User Details";
+    public static final String ENTER_FULLNAME = "1. Enter Fullname";
+    public static final String ENTER_DEPARTMENT = "2. Enter Department";
+    public static final String ENTER_ADDRESS = "3. Enter Address";
+    public static final String ENTER_USERNAME = "4. Enter Username";
+    public static final String ENTER_PASSWORD = "5. Enter Password";
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Remove this hard-coded password. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS2068)